5/README.md

```diff
@@ -22,46 +28,76 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
-
-* Provide the initial information when one starts a Tango Device
-
-* Be usable as keeping global variable in the Tango control system
-
-* Provide meta data complementing any data in Tango
+There are 4 kinds of Tango Properties:
 
+ *  Class Property: a Property related to a Device Class (cf RFC-2).
+ *  Device Property: a Property related to a Device (RFC-2).
+ *  Attribute Property: a Property related to an Attribute (RFC-4).
+ *  Free Property: a Property defined for the entire Control System (RFC-1) i.e not related to any Class, Device or Attribute.
 
 ### Use Cases
 
-There are many use cases for the usage of a Property:
-
-* To connect a real network equipment a Property can define the target IP address
-
-* To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data.
+Here are some Tango Property use cases:
 
+* To connect a real network equipment a Property can define the target IP address or hostname
+* To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data
+* To configure some alarms on Attribute Value (See RFC-4)
+* To store the Attribute value of a Memorized Attribute (See RFC-4)
+* To configure the labels associated to a Tango:DevEnum attribute values
+* ...
 
 ## Specification
 
-A Tango Property is a strict definition of a pair of key/value
-* configuration are represented in the form of properties.
-* data are represented in the form of attributes.
-* actions are represented in the form of commands.
-
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
-
-
-A Tango Property is a strict definition of a pair of key/value
-* The Property SHALL have one key, called Property Name
-* The Property SHALL have one value
+A Tango Property is a strict definition of a pair of key/value:
+  * The Property SHALL have one key, called Property Name
+  * The Property SHALL have one value, which could be empty, called Property Value
+
+A Tango Property MAY be persisted in the Tango Database of the concerned Tango Control System (See RFC-1 and RFC-6) or into a file if no Tango Database is used.
+
+Tango Device Properties and Tango Class Properties have a Type metadata.
+The following Types SHALL be supported by Device Properties and Class Properties (See RFC-9):
+  * DevBoolean
+  * DevShort
+  * DevUShort
+  * DevLong
+  * DevULong
+  * DevLong64
+  * DevULong64
+  * DevFloat
+  * DevDouble
+  * DevString
+  * DevVarShortArray
+  * DevVarLongArray
+  * DevVarLong64Array
+  * DevVarFloatArray
+  * DevVarDoubleArray
+  * DevVarStringArray
+
+A conforming Tango implementation SHALL provide a way to retrieve a persisted Property Value and to convert it to the above types at least.
```

Comment:
Suggest to remove "at least"

Reply:
The idea behind this " at least" was to avoid to limit ourselves to these types.
As a device property is a configuration parameter, we can imagine extending this in the future with some other types very useful in the astronomy world for instance, like images.
But I agree with you, we should define this list to ensure the clients and servers can both support all the types. If we add the support for images, it would be useful for generic tools like jive to be able to display these images.
If we want to add new types, we can update the specs in the future...
I'll follow your suggestion.

Reply:
Done in https://github.com/tango-controls/rfc/pull/59/commits/898755d7f07eb6dfdf5bcb2fe56fbb2de7d394c4